org.ferdium.Ferdium.yaml

```diff
@@ -0,0 +1,129 @@
+rename-icon: ferdium
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --filesystem=home:ro
+  - --filesystem=xdg-download
+  - --filesystem=/run/.heim_org.h5l.kcm-socket
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.*
+
+# Anchors
+.build-commands: &build
+    - ar x ferdium_*.deb
+    - rm -f ferdium_*.deb
+    - tar xf data.tar.xz
+    - rm -f control.tar.gz data.tar.xz debian-binary
+    - cp -r usr/* opt/* /app
+    - chmod -R a-s,go+rX,go-w /app/Ferdium
+    - rm -r /app/share/icons/hicolor/1024x1024
+    - desktop-file-edit --set-key=Exec --set-value='ferdium %U' /app/share/applications/ferdium.desktop
+    - install ferdium.sh /app/bin/ferdium
+    - install -Dm644 org.ferdium.Ferdium.metainfo.xml /app/share/metainfo/org.ferdium.Ferdium.metainfo.xml
+.sources:
+  - &source
+    type: file
+    url: https://github.com/ferdium/ferdium-app/archive/refs/tags/v6.0.0-beta.1.tar.gz
+    sha256: dcb8ecc5e0357bf81551120ca9efe5e95368bcaaa60b22fa4750a4edc5d22e9d
+    x-checker-data:
+      type: html
+      url: https://ferdium.org/download/
+      version-pattern: 'Current version: ([\d\.-]+(\-beta\.\d+))?'
+      url-template: https://github.org.ferdium/ferdium-app/archive/refs/tags/v$version.tar.gz
+  - &script
+    type: script
+    dest-filename: ferdium.sh
+    commands:
+      - export TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID"
+      - exec zypak-wrapper /app/Ferdium/ferdium "$@"
+  - &metainfo
+    type: file
+    path: org.ferdium.Ferdium.metainfo.xml
+
+modules:
+  - name: krb5
+    subdir: src
+    config-opts:
+      - --disable-static
+      - --disable-rpath
+    sources:
+      - type: archive
+        url: https://kerberos.org/dist/krb5/1.19/krb5-1.19.3.tar.gz
+        sha256: 56d04863cfddc9d9eb7af17556e043e3537d41c6e545610778676cf551b9dcd0
+        x-checker-data:
+          type: anitya
+          project-id: 13287
+          stable-only: true
+          url-template: https://kerberos.org/dist/krb5/$major.$minor/krb5-$major.$minor.$patch.tar.gz
+  - name: krb5-config
+    buildsystem: simple
+    sources:
+      - type: file
+        path: krb5.conf
+    build-commands:
+      - mkdir -p /app/etc
+      - install -m 644 krb5.conf /app/etc/krb5.conf
+  - name: ferdium-x86_x64
+    buildsystem: simple
+    build-commands: *build
+    only-arches:
+      - x86_64
+    sources:
+      - type: file
+        url: https://github.com/ferdium/ferdium-app/releases/download/v6.0.0-beta.1/ferdium_6.0.0-beta.1_amd64.deb
+        sha256: 89445c022abae41341ea9c6963e5a56b53b08391432c11bde125900998cc4f50
+        x-checker-data:
+          type: html
+          url: https://ferdium.org/download/
+          version-pattern: 'Current version: ([\d\.-]+(\-beta\.\d+))?'
+          url-template: https://github.org.ferdium/ferdium-app/releases/download/v$version/ferdium_${version}_amd64.deb
+      - *source
```

Comment:
the trick here would be to have ONE module, and mark each source as `only-arches`. And avoid these "referemces" that make everything hard to follow. Bonus point: the manifest is way shorter.

Reply:
Oh neat. I was experimenting with anchors and noticed that merge keys didnt work either so I stopped right there. Will update it real quick